tasks/README.md

```diff
@@ -228,4 +228,7 @@ Name | Summary | Category
 `task273_europarl_classification.json`| Given bulgarian sentence and corresponding english translation, verify that the translation is right or wrong | Classification
 `task274_overruling_classification`	| Given a sentence, classify it into overruling or non-overruling| Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect | Text Modification
-`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
\ No newline at end of file
+`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task309_RACE_answer_generation` | Given an article, a question and four options; provide correct answer for the question based on the article. | Answer Generation
```

Comment:
we gotta lowercase these names and merge with the main branch.

Reply:
Done.